Gemfile.lock

```diff
@@ -78,17 +78,17 @@ GEM
       regexp_parser (~> 1.5)
       xpath (~> 3.2)
     childprocess (3.0.0)
-    concurrent-ruby (1.1.5)
-    crass (1.0.5)
+    concurrent-ruby (1.1.6)
+    crass (1.0.6)
     erubi (1.9.0)
     execjs (2.7.0)
-    ffi (1.11.3)
+    ffi (1.12.2)
     globalid (0.4.2)
       activesupport (>= 4.2.0)
-    i18n (1.7.0)
+    i18n (1.8.2)
       concurrent-ruby (~> 1.0)
-    jbuilder (2.9.1)
-      activesupport (>= 4.2.0)
+    jbuilder (2.10.0)
```

Comment:
### [jbuilder](https://github.com/rails/jbuilder)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.9.1 → 2.10.0

[_~~change-log~~, [source-code](https://github.com/rails/jbuilder)_]





<details>
<summary>Commits</summary>

A change of **9** commits. See the full changes on [the compare page](https://github.com/rails/jbuilder/compare/v2.9.1...v2.10.0).

These are the individual commits:
- (2bb1eac) [Test against Rails 6.0.0 final](https://github.com/rails/jbuilder/commit/2bb1eac02c50279a33164766f71e7f11f90e3d19)
- (44bc207) [Require Rails 5+ and Ruby 2.2+](https://github.com/rails/jbuilder/commit/44bc207ff15fb9f0c1a58138b90122c0d300dff3)
- (a2cf56e) [Update comment in controller generator templates](https://github.com/rails/jbuilder/commit/a2cf56ed20fd9178c61b593ec0c9364256bfe0b9)
- (fd65c71) [Merge pull request #467 from eileencodes/change-comment-in-c…](https://github.com/rails/jbuilder/commit/fd65c7140d3142270d5723c6294c4c6cf035cadf)
- (454d697) [Fix warnings on Ruby 2.7](https://github.com/rails/jbuilder/commit/454d69729ffbde221117992a805f70d3a39b5ba4)
- (e5f57cf) [Merge pull request #468 from r7kamura/feature/fix-ruby-2-7-w…](https://github.com/rails/jbuilder/commit/e5f57cff32275afda17ac1674ff79f0451e5ce40)
- (5e0aa27) [Deep-merge nested hashes](https://github.com/rails/jbuilder/commit/5e0aa272ff1943db42776cecbf951c809ac29110)
- (bec0d6c) [Move require](https://github.com/rails/jbuilder/commit/bec0d6c840c3486ee589d5de0ba9c348ccbc27ee)
- (e0ac4a1) [v2.10.0](https://github.com/rails/jbuilder/commit/e0ac4a1f2fa3d22f9f3bb94194ed499ca91cf40e)


</details>